Lecture3/Task1/Voucher.java

```diff
@@ -0,0 +1,41 @@
+abstract class Voucher {
+    private String name;
+    private double price;
+    private String country;
+    private String startDate;
+    private int countDays;
+    private int discount;
+    private double cost;
+
+    int getCountDays() {
+        return countDays;
+    }
+
+    double getCost() {
+        return cost;
+    }
+
+    public abstract String getFood();
+
+    public abstract String getTransfer();
+
+    public void initVoucher(int i) {
+        this.name = "Name" + i;
+        this.price = i;
+        this.country = "Country" + i;
+        this.startDate = "StartDate" + i;
+        this.countDays = i;
+        this.discount = i;
+        this.cost = this.price - this.price * this.discount / 100;
+    }
+
+    public void printVoucher() {
```

Comment:
printVoucher method can be replaced with toString() method. Then all you need is just to System.out.println(voucher);

Reply:
fixed